src/main/java/seedu/duke/common/Messages.java

```diff
@@ -1,4 +1,5 @@
 package seedu.duke.common;
 
 public class Messages {
+    public static final String ADDED_LESSON_MESSAGE = "Added: ";
```

Comment:
Might want to call it MESSAGE_ADDED_LESSON instead, so all messages will have the same prefix.

Reply:
Will change it the constant name